src/pages/base-page.ts

```diff
@@ -25,6 +26,10 @@ export class BasePage {
 
 
     // Search by single element
+    public bySystelabTestId(id: string): ElementFinder {
+        return this.current.byId(id);
```

Comment:
This should not call the byId() method but the bySystelabTestId() one, also better use dataTestId as parameter

Reply:
True, thanks. I left the copy-paste and forgot to change the body like for the rest

Reply:
I still see some new methods using arguments referring to "data", so I would insist in hiding this implementation detail from the interface.